localization/languages/cs-CZ.json

```diff
@@ -0,0 +1,152 @@
+        "crashErrorSubtitle": "Během zobrazování této stránky došlo k potížím.",
+        /* pages/error/index.html */
+        "errorPagePrimaryAction": "Zkusit znovu",
+        "serverNotFoundTitle": "Server nenalezen",
+        "serverNotFoundSubtitle": "Prohlížeč Min nemůže najít tuto webovou stránku.",
+        "archiveSearchAction": "Vyhledávat na archive.org",
+        "sslErrorTitle": "Tato webová stránka není dostupná",
+        "sslErrorMessage": "Prohlížeč Min nemůže s touto stránkou navázat zabezpečené spojení.",
+        "dnsErrorTitle": "Webová stránka nenalezena",
+        "dnsErrorMessage": "Došlo k chybě DNS.",
+        "offlineErrorTitle": "Jste offline",
+        "offlineErrorMessage": "Připojte se k internetu a zkuste to znovu.",
+        "genericConnectionFail": "Prohlížeč Min se nemůže připojit k této webové adrese.",
+        "sslTimeErrorMessage": "Prohlížeč Min nemůže s touto stránkou navázat zabezpečené spojení. Prosím zkontrolujte, zda máte v počítači správně nastavený čas.",
+        "addressInvalidTitle": "Tato adresa je neplatná.",
+        "genericError": "Vyskytla se chyba",
+        /* pages/phishing/index.html */
+        "phishingErrorTitle": "Tyto stránky by vám mohly ublížit.",
+        "phishingErrorMessage": "Tyto webové stránky se mohou pokusit ukrást vaše osobní údaje, například hesla nebo bankovní informace.",
+        "phishingErrorVisitAnyway": "Přesto stránky navštívit",
+        "phishingErrorLeave": "Opustit tyto stránky",
+        /* pages/sessionRestoreError/index.html */
+        "sessionRestoreErrorTitle": "Vyskytla se chyba",
+        "sessionRestoreErrorExplanation": "Vaše uložené karty nelze správně obnovit.",
+        "sessionRestoreErrorBackupInfo": "Zálohu vašich dat jsme uložili na tomto místě: %l.", //%l will be replaced with a path to a file
+        "sessionRestoreErrorLinkInfo": {
+            "unsafeHTML": "Pokud se tato chyba opakuje, založte prosím nový problém <a href=\"https://github.com/minbrowser/min\" target=\"_blank\">zde</a>."
+        },
+        /* pages/settings/index.html */
+        "settingsPreferencesHeading": "Nastavení",
+        "settingsRestartRequired": "Pro použití změn musíte restartovat aplikaci.",
+        "settingsPrivacyHeading": "Blokování obsahu",
+        "settingsContentBlockingToggle": "Blokovat trackery a reklamy",
+        "settingsBlockScriptsToggle": "Blokovat skripty",
+        "settingsBlockImagesToggle": "Blokovat obrázky",
+        "settingsEasyListCredit": {
+            "unsafeHTML": "Na základě EasyList a EasyPrivacy bez pravidel skrývání prvků. Vytvořili <a target=\"_blank\" href=\"https://easylist.to/\">The EasyList authors</a> - <a target=\"_blank\" href=\"https://creativecommons.org/licenses/by-sa/3.0/legalcode\">s licencí</a>."
+        },
+        "settingsAppearanceHeading": "Vzhled",
+        "settingsDarkModeToggle": "Použít tmavý režim",
+        "settingsSwipeNavigationHeading": "Swipe navigation",
```

Comment:
Is is possible to translate this and the string below it?

Reply:
Can you please explain me, what is this feature for? I can't get this to work on Linux so I don't know, how to correctly translate this.

Reply:
It lets you place two fingers on your trackpad, and use a swiping gesture to navigate back and forwards, like this: https://youtu.be/6NBMHEWLSoc?t=95.

Reply:
Ahh, I see! Ok, I'll try translate it.